subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/matcher/RecyclerViewMatcher.kt

```diff
@@ -48,6 +49,15 @@ class RecyclerViewMatcher {
         }
     }
 
+    fun hasViewTypeAtPosition(position: Int, viewType: Int): Matcher<View> = object : RecyclerViewMatcher() {
```

Comment:
Let's make anonymous object separate class or file

Reply:
The idea was that we want to check any parameter of view, not only the `viewType`
I want to see hasView(position, matcher)